### PR TITLE
Decoupled Input Output with operation files.

### PR DESCRIPTION
**Pull request by sahityakmr**

### CODE REVIEW

File: src/com/akmr/sm/StudentManager.java [hunk 1]
```diff
@@ -1,292 +1,133 @@
 package com.akmr.sm;
 
 import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Scanner;
```
Comment (anmolkmr) [inline src/com/akmr/sm/StudentManager.java:5]: Why did you remove line number 4 and 5.

Comment (sahityakmr) [reply src/com/akmr/sm/StudentManager.java:5]: They are no more required.

**Review by anmolkmr — APPROVED** [review]
